depends/check-dependencies.sh

```diff
@@ -54,7 +80,7 @@ if [ "$(uname)" == "Darwin" ]; then
 fi
 
 check_header    libelf          elf.h libelf.h libelf/libelf.h gelf.h libelf/gelf.h
-check_header    libusb          usb.h
+check_header    libusb          libusb.h
```

Comment:
This is probably not a good idea. We need either libusb 1 or libusb-compat, which both ship ``usb.h``. The code will not build with newer libusb versions.

Reply:
I changed it to check for `usb.h` header and then for `libusb.h` both advanced with pkg-config.
It returns the first matched of the header names listed in `check_header libusb usb.h libusb.h`.

Reply:
Which program is using libusb.h to build?

Reply:
gentoo version of libusb-1.0.24-r1 uses libusb-1.0/libusb.h
gentoo version of libusb-compat-0.1.5-r3 uses libusb-compat/usb.h
the psp dependency is psplinkusb and probably more.